packages/user-mgnt/src/features/createUser/service.ts

```diff
@@ -111,14 +100,6 @@ export const createFhirPractitionerRole = async (
               code: user.systemRole
             }
           ]
-        },
-        {
-          coding: [
```

Comment:
I wasn't able to immediately find if these are used or if they are used at all. I decided to remove these as end user friendly text labels shouldn't be stored in the database but we should use role ids and message objects instead